samples/manager/package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Same kind of question as above - can we combine these into a single `package.json` in `samples/`?

Reply:
Digging back into this... the IoT samples are distinct, atomic samples:

* Manager - uses this client library
* MQTT Client - uses PAHO MQTT client and manager sample to bootstrap tests
* HTTP Client - uses HTTP request to make curl-like API calls and also uses manager to bootstrap tests

are you sure we should group all of the dependencies across the distinct samples?  It may logically make sense to remove the HTTP/MQTT samples from here as they will still need to exist in the monorepo until we can figure out how to extract snippets from the client library repo for docs.

Reply:
I think all the tests will run as configured and they also should still be working atomically.